src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -240,6 +250,6 @@ def app_deploy(
     )
 
     manager.build_bundle()
-    manager.deploy()
+    manager.deploy(prune, files)
```

Comment:
this is what I meant, isn't this then making deploy without args not prune existing files? That's not what is intended.